test/fuzz/lua/test_mvcc.lua

```diff
@@ -0,0 +1,872 @@
+                                  {is_nullable = true}}),
+            create_tree_idx_meta({{is_nullable = true},
+                                  {is_nullable = true}}),
+            create_tree_idx_meta({{is_nullable = false},
+                                  {is_nullable = true}}),
+            create_hash_idx_meta({{is_nullable = false},
+                                  {is_nullable = false}}),
+            create_tree_idx_meta({{is_nullable = false},
+                                  {is_nullable = false}}),
+        }
+    },
+    {
+        name = 'vinyl',
+        space = 'v',
+        idxs = {
+            create_tree_idx_meta({{is_nullable = false},
+                                  {is_nullable = false}}),
+            create_tree_idx_meta({{is_nullable = true},
+                                  {is_nullable = false}}),
+            create_tree_idx_meta({{is_nullable = false},
+                                  {is_nullable = true}}),
+            create_tree_idx_meta({{is_nullable = true},
+                                  {is_nullable = true}}),
+            create_tree_idx_meta({{is_nullable = true},
+                                  {is_nullable = true}}),
+            create_tree_idx_meta({{is_nullable = false},
+                                  {is_nullable = true}}),
+        }
+    },
+}
+
+-- Global test variables.
+
+-- File for dumping the executed transactions for the current round.
+local repro_file
+-- File for dumping the serial transaction execution schedule for the current
+-- round.
+local serial_file
+-- List of all transaction statements for the current round.
+local stmts
+-- Mask of read-only transactions for the current round.
+local ro_txs_mask
+-- Mask of transactions that got successfully committed during the current
+-- round.
+local committed_txs_mask
+-- Mask of transactions that had an erroneous DML operation for the current
+-- round.
+local bad_dml_txs_mask
+-- Serial schedule of transactions for the current round.
+local serial
+-- Server used for testing MVCC.
+local mvcc = nil
+-- Oracle server used for checking MVCC consistency.
+local oracle = nil
+
+local function generate_txn_repro_stmt(stmt)
+    return ('tx%d(\'%s\') -- %s\n'):format(stmt.tid, stmt.str,
+                                         json.encode(stmt.res))
+end
+
+local function generate_txn_serial_stmt(stmt)
+    return ('%s -- tx%d: %s\n'):format(stmt.str, stmt.tid,
+                                       json.encode(stmt.res))
+end
+
+local INIT_CODE = [[
+os.execute('rm -rf *.snap *.xlog *.vylog 512 513')
+
+local ffi = require('ffi')
+local fiber = require('fiber')
+local json = require('json')
+local log = require('log')
+local txn_proxy = require('test.box.lua.txn_proxy')
+
+box.cfg{memtx_use_mvcc_engine = %s}
+fiber.set_max_slice({warn = 6000, err = 6000})
+
+]]
+
+local function generate_init_code(memtx_use_mvcc_engine)
+    return (INIT_CODE):format(memtx_use_mvcc_engine and 'true' or 'false')
```

Comment:
`memtx_use_mvcc_engine` is already a boolean
s/memtx_use_mvcc_engine and 'true' or 'false'/memtx_use_mvcc_engine/